### PR TITLE
Update asset importing spec to use `import.meta.resolve`

### PR DESCRIPTION
**Pull request by kategengler**

### CODE REVIEW

File: text/0763-asset-importing-spec.md [hunk 1]
```diff
@@ -86,20 +94,51 @@ HTML | Other | `<img src="./thing.png" />` | W3C
...
+    return import.meta.resolve(`../icons/${this.args.locale}.png`)
+  }
+}
+```
+
+Calling `import.meta.resolve` with an asset that does not exist will cause a build error. You cannot fail to notice if you accidentally delete an asset that is still in use somewhere in your app.
+
+#### Isn't it risky to "take over" a browser API?
+
+Yes and no. This [original proposal for this feature to WHATWG](https://github.com/whatwg/html/issues/3871#issue-346547968) envisioned build tools rewriting the expressions to provide asset optimizations.
```
Comment (simonihmig) [inline text/0763-asset-importing-spec.md:123]: What's the risk that "Yes" is referring to? I read the following text as arguing against the existence of risk, or do I misunderstand this?

Comment (kategengler) [reply text/0763-asset-importing-spec.md:123]: No, just poor wording.

Comment (kategengler) [inline text/0763-asset-importing-spec.md:123]:
```suggestion
No. This [original proposal for this feature to WHATWG](https://github.com/whatwg/html/issues/3871#issue-346547968) envisioned build tools rewriting the expressions to provide asset optimizations. As describe above, it is only taken over for "Other Assets" (any path that ends in an explicit file extension that is not `.js` or `.css`). 
```

File: text/0763-asset-importing-spec.md [hunk 2]
```diff
@@ -69,15 +77,15 @@ From Type | To Type | Example Syntax | Semantics
...
 CSS | CSS | `@import("thing/style.css")` | [W3C](https://drafts.csswg.org/css-cascade/#at-import) plus NodeJS resolving rules
 CSS | Other | `url("thing/icon.png");` | W3C plus NodeJS resolving rules
 HTML | JS | `<script src="./thing.js"></script>` | W3C
 HTML | CSS | `<link rel="stylesheet" href="./thing.css"/>` | W3C
 HTML | Other | `<img src="./thing.png" />` | W3C
 
 
-1. Importing a file with an explicit `.css` extension guarantees that the given CSS will be loaded into the DOM before your module executes. We do not define any exported values, this is purely for side-effect. This rule is not part of the present RFC, it was already in RFC 507.
+1. Importing a file with an explicit `.css` extension guarantees that the given CSS will be loaded into the DOM before your module executes. We do not define any exported values, this is purely for side-effect. This rule is not part of the present RFC, it was already in [RFC 507](https://github.com/emberjs/rfcs/blob/master/text/0507-embroider-v2-package-format.md).
 2. "Other Asset" means this rule applies to any JS import of a path that ends in an explicit file extension that is not `.js` or `.css`, meaning:
```
Comment (kategengler) [inline text/0763-asset-importing-spec.md:89]:
```suggestion
2. "Other Asset" means this rule applies to any `import.meta.resolve` with a path that ends in an explicit file extension that is not `.js` or `.css`, meaning:
```

**Review by ef4 — APPROVED** [review]

**Comment by ef4** [comment]
Thanks, merging these changes into https://github.com/emberjs/rfcs/pull/763

**Pull request merged**